.dockerignore

```diff
@@ -0,0 +1 @@
+pika/node_modules
```

Comment:
We should ignore *all* node_modules not only the ones in a specific subfolder (merge in master)

Reply:
Because we install them in the Docker image and don't need to load them from the environment where the script is executed